### DISCUSSION TITLE
Scheduled Actions Not Triggered

### DISCUSSION THREAD

**Discussion by ries9112** [community/community#25420] [Actions]
<p>Hello, I have a repo that has so far executed over 2,400 automated runs on an automated hourly schedule with no issues: <a href="https://github.com/ries9112/ries9112/actions" rel="noopener nofollow ugc">https://github.com/ries9112/ries9112/actions</a></p>
<p>Yesterday I applied some changes to cache the packages and avoid installing things every time to get the runtime from ~10min down to ~2min. It all works exactly how I want it to, but now GitHub Actions decided to stop working on a schedule… I feel like I’ve tried everything like creating new workflows to try and get things “unstuck” which is what I was finding online, but doesn’t seem to have done the trick.</p>
<p>Sometimes they randomly appear, when I push stuff, but I have no clue why they aren’t showing up, can someone please help?? I created an example with 4 different workflows to see if maybe it was a change I made to the .yml file, but none of them are being triggered: <a href="https://github.com/ries9112/ries9112-test/tree/master/.github/workflows" rel="noopener nofollow ugc">https://github.com/ries9112/ries9112-test/tree/master/.github/workflows</a></p>
<p>Thoughts on what the heck is going on?</p>

**Comment by airtower-luna** [comment]
<p>For the <code>new_test*</code> files I’d guess they aren’t recognized as workflow files because their names don’t end with <code>.yaml</code> or <code>.yml</code>.</p>
<p>I’m not sure which might be disturbing the scheduling, the schedule setting looks correct to me. There is a warning about the build using an outdated macOS version, but I don’t see how that would disturb the scheduling itself.</p>

**Comment by ries9112** [comment, marked as answer]
<p>Thanks for the response. Things seem to have fixed themselves now, no clue what was going on earlier. Made no change to my .yml file, now working. Also thanks about the note regarding the warning, the version I wanted was actually macOS-10.15 not 10.14, now working beautifully without warnings: <a href="https://github.com/ries9112/ries9112/actions" rel="noopener nofollow ugc">https://github.com/ries9112/ries9112/actions</a></p>
<p>The runs seem to be kicking off around the 20th+ minute of the hour instead of the 10th minute, but as long as it’s running I’m happy haha.</p>
